db/seeds/utils.js

```diff
@@ -34,4 +35,13 @@ exports.getCommentById = (comment_id) => {
     } else {
     return result.rows[0]}
   })
+}
+
+exports.doesTopicExist = (targetTopic) => {
+  for (const topic of topics) {
+    if (topic.slug === targetTopic){
+      return true
+    } 
```

Comment:
For this you should make a query to the database - you are only checking if the topic exists in the test database at the moment but when you host it will use the dev data - and also what if someone adds a new topic? 

SELECT * from topics will cover you for any set of data in any database.